website/docs/en-US/time-select.md

```diff
@@ -69,7 +69,8 @@ If start time is picked at first, then the end time will change accordingly.
 ### Attributes
 | Attribute      | Description          | Type      | Accepted Values       | Default  |
 |---------- |-------------- |---------- |--------------------------------  |-------- |
-| value / v-model | binding value | string | - | - |
+| value / v-model | binding value | string | — | — |
```

Comment:
`value` in Vue 3 is replaced by `modelValue`, can you please help changing these?